app/liquid_tags/gist_tag.rb

```diff
@@ -32,6 +32,10 @@ def build_uri(link)
   end
 
   def parse_link(link)
+    if link.nil?
```

Comment:
We can perform this check alrady in the initializer, e.g. 

```ruby
raise StandardError, "Invalid Gist link: You must provide a Gist link" if link.blank?
```

There's no reason to even attempt the things in `build_uri` if link is `""`.

```